package.json

```diff
@@ -40,11 +40,10 @@
   ],
   "license": "MIT",
   "dependencies": {
```

Comment:
I was wondering to change it to peerDependencies instead. What do you think @zenorocha?